build-tools/create-packs/Microsoft.Android.Sdk.proj

```diff
@@ -118,18 +118,23 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <PropertyGroup>
     <AndroidNETSdkVersion>$(AndroidPackVersionLong)</AndroidNETSdkVersion>
     <XamarinAndroidVersion>$(AndroidPackVersionLong)</XamarinAndroidVersion>
-    <_AndroidRuntimePackId Condition=" '%24(TargetPlatformVersion)' != '$(AndroidLatestUnstableApiLevel).0' ">$(AndroidLatestStableApiLevel)</_AndroidRuntimePackId>
-    <_AndroidRuntimePackId Condition=" '%24(TargetPlatformVersion)' == '$(AndroidLatestUnstableApiLevel).0' ">$(AndroidLatestUnstableApiLevel)</_AndroidRuntimePackId>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '%24(TargetPlatformVersion)' == '31.0' ">
```

Comment:
Should this be using `$(AndroidTargetDotNetApiLevel)` (and possibly other new properties)? Isn't that what `$(AndroidTargetDotNetApiLevel)` is for?

Related: instead of `$(AndroidTargetDotNetApiLevel)`, should we call it `$(AndroidDefaultTargetDotnetApiLevel)`?  This "emphasizes" the "default" nature of this value.

Reply:
In some of the other tests here, I'll try to use `$(AndroidDefaultTargetDotnetApiLevel)` so we don't have to change them again later.

In this case, however, the number 31 needs to remain, it will need to eternally pull this version from NuGet (or our workload):

https://github.com/xamarin/xamarin-android/blob/bedecc25f93a82ecaa8df5cb4e92424aa669125d/build-tools/create-packs/Microsoft.Android.Sdk.proj#L122-L125

I guess that over time, we'll have numbers listed for 32, 33, and so on. We can also update those packs at some point if ever needed (hopefully not ever?), we'd just list a different version number here.